tests/api2/test_005_interface.py

```diff
@@ -60,8 +63,17 @@ def get_payload(ws_client):
 
     return payload, to_validate
 
+# Make sure that our initial conditions are met
+def test_001_check_ipvx(request, ws_client, get_payload):
+    # Verify that dhclient is running
+    ps_count = int(SSH_TEST('ps -aux | grep dhclient | wc -l', user, password, truenas_server.ip)['stdout'])
+    assert ps_count > 1 # account for the grep
 
-def test_001_configure_interface(request, ws_client, get_payload):
+    # Check that our proc entry is set to its default 1.
+    autoconf = int(SSH_TEST(f'cat /proc/sys/net/ipv6/conf/{interface}/autoconf', user, password, truenas_server.ip)['stdout'])
```

Comment:
I don't think there is any need to use `SSH_TEST` here, we can get the same effect with
```
midclt call tunable.get_sysctl 'net.ipv6.conf.br0.autoconf'
```